spec/marten/template/tag/for/loop_spec.cr

```diff
@@ -101,4 +101,32 @@ describe Marten::Template::Tag::For::Loop do
       loop.revindex0.should eq 8
     end
   end
+
+  describe "#odd?" do
+    it "returns true if the index is odd" do
+      loop = Marten::Template::Tag::For::Loop.new(items_size: 10)
+      loop.index = 1
+      loop.odd?.should eq true
+
+      loop.index = 2
+      loop.odd?.should eq false
```

Comment:
I would expect this to be in another spec example (eg. `returns false if the index is not odd`).